backend/benefit/applications/tests/conftest.py

```diff
@@ -356,5 +355,5 @@ def pytest_sessionfinish(session, exitstatus):
 
 
 @pytest.fixture()
-def ahjo_open_case_payload(decided_application):
-    return prepare_open_case_payload(decided_application)
+def ahjo_open_case_payload():
+    return {"foo": "bar"}
```

Comment:
foo bar?

Reply:
This can be anything, as long as it's a dict,as the test that uses it, does not perform an actual Ahjo request, but tests different failed responses of a  mocked ahjo request, I guess I could remove this fixture.